src/lesson7/javaBeginnerChapter2/task27/CashCard.java

```diff
@@ -0,0 +1,7 @@
+package lesson7.javaBeginnerChapter2.task27;
+
+abstract class CashCard extends BankCard {
+    public CashCard(String name) {
```

Comment:
The same as previous comment - you should extend superclass, but not duplicate it

Reply:
fixed